pyvocz/templates/fragments/series-pagination.html

```diff
@@ -0,0 +1,22 @@
+<div>
+    <ul class="paginate">
+        <li class="paginate-prev">
+            <a {% if year==None %}class="disabled"{% endif %}
+               href="{{ url_for('series', series_slug=series_slug,
+                                year=None if year==None or year>=last_year else year+1) }}">&lt;</a>
+        </li>
+        <li class="paginate-link {% if year==None %}current{% endif %}">
+             <a href="{{ url_for('series', series_slug=series_slug, year=None) }}">{{ tr("Nové", "New") }}</a>
+        </li>
+        {% for page in range(last_year, first_year-1, -1) %}
+            <li class="paginate-link {% if page==year %}current{% endif %}">
+                 <a href="{{ url_for('series', series_slug=series_slug, year=page) }}">{{ page }}</a>
+            </li>
+        {% endfor %}
+        <li class="paginate-next">
+            <a {% if year!=None and year<=first_year %}class="disabled"{% endif %}
+               href="{{ url_for('series', series_slug=series_slug,
+                                year=last_year if year==None else year-1) }}">&gt;</a>
+        </li>
+    </ul>
+</div>
```

Comment:
Please use a local macro instead. In `series.html`, you can write:

```
{% macro pagination() %}
<div>
    ...
</div>
{% endmacro %}
```

and then use it as `{{ pagination() }}`.

The "fragments" are an embarassing artefact from a time when I didn't know about macros. (I wrote commits to get rid of them, but didn't submit them yet as I don't want to conflict with this work.)